### ISSUE TITLE
Is it possible to use a tv instead of pagetitle in the list?

### ISSUE THREAD

**Issue by em-piguet** [benjamin-vauchel/SuperBoxSelect#3]
My setup : 
I have a folder full of resources containing artists.
for example, my pagetitle is "Dylan". But I have a tv "fullname" in my resource with a value of "Bob Dylan". I would like to use this tv in the superboxselect instead of the pagetitle.. I know it's a big enhancement but i think it will be very useful :-)

**Comment by benjamin-vauchel** [comment]
You're right. It's an essential feature. The next version for MODx Revo 2.2 will come with input options and it will be possible to specify the query field.
